osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -100,7 +101,16 @@ private void onPathChanged(ValueChangedEvent<string> e)
             cancellationToken?.Cancel();
             request?.Cancel();
 
-            request = new GetWikiRequest(e.NewValue);
+            string[] values = e.NewValue.Split('/', 2);
+
+            if (values.Length > 1 && LanguageExtensions.TryParseCultureCode(values[0], out _))
+            {
+                request = new GetWikiRequest(values[1], values[0]);
+            }
+            else
+            {
+                request = new GetWikiRequest(e.NewValue);
+            }
```

Comment:
Looking at `GetWikiRequest`, wouldn't it be simpler to just make `GetWikiRequest` not require a `locale` parameter and not require this parsing? i.e.:
```diff
diff --git a/osu.Game/Online/API/Requests/GetWikiRequest.cs b/osu.Game/Online/API/Requests/GetWikiRequest.cs
index 248fcc03e3..fbd247d17a 100644
--- a/osu.Game/Online/API/Requests/GetWikiRequest.cs
+++ b/osu.Game/Online/API/Requests/GetWikiRequest.cs
@@ -8,14 +8,12 @@ namespace osu.Game.Online.API.Requests
     public class GetWikiRequest : APIRequest<APIWikiPage>
     {
         private readonly string path;
-        private readonly string locale;

-        public GetWikiRequest(string path, string locale = "en")
+        public GetWikiRequest(string path)
         {
             this.path = path;
-            this.locale = locale;
         }

-        protected override string Target => $"wiki/{locale}/{path}";
+        protected override string Target => $"wiki/{path}";
     }
 }
```

Reply:
That will probably be fine until we want to use the user's local set language and need to pass it in.

Reply:
Right, I think going with your direction sounds better.

Reply:
Keep in mind that we *may* address this via an http header, I'm not 100% sure yet. But the extra flexibility is probably not a bad thing (having it available in the request).

Reply:
Being able to link to translations of an article is definitely a needed feature, where it would override any http header or locale setting (as it already does).

It's mainly for reference purposes in the case of translations (or just for linking a translation for those who don't set their locale), and being able to guarantee that the recipient will receive the latest version of the article.